src/ui/components/Sidebar.tsx

```diff
@@ -1,28 +1,54 @@
 import { FC } from "react";
 import { Link } from "react-router-dom";
+import { Nav } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
 
 const Sidebar: FC = () => {
   return (
-    <div className="sidebar">
-      <Link to="/experiment-json" className="sidebar__link">
+    <Nav className="sidebar rounded position-fixed m-2 light-background">
+      <Nav.Link
+        as={Link}
+        to="/experiment-json"
+        className="sidebar__link fw-medium p-4 dark-font"
+      >
         JSON Enrollment
-      </Link>
-      <Link to="/experiment-feature-config" className="sidebar__link">
+      </Nav.Link>
+      <Nav.Link
+        as={Link}
+        to="/experiment-feature-config"
+        className="sidebar__link fw-medium p-4 dark-font"
+      >
         Feature Configuration Enrollment
-      </Link>
-      <Link to="/jexl-debugger" className="sidebar__link">
+      </Nav.Link>
+      <Nav.Link
+        as={Link}
+        to="/jexl-debugger"
+        className="sidebar__link fw-medium p-4 dark-font"
+      >
         JEXL Debugger
-      </Link>
-      <Link to="/experiment-store" className="sidebar__link">
+      </Nav.Link>
+      <Nav.Link
+        as={Link}
+        to="/experiment-store"
+        className="sidebar__link fw-medium p-4 dark-font"
+      >
         Experiment Store
-      </Link>
-      <Link to="/experiment-browser" className="sidebar__link">
+      </Nav.Link>
+      <Nav.Link
+        as={Link}
+        to="/experiment-browser"
+        className="sidebar__link fw-medium p-4 dark-font"
+      >
         Experiment Browser
-      </Link>
-      <Link to="/settings" className="sidebar__link">
+      </Nav.Link>
+      <Nav.Link
+        as={Link}
+        to="/settings"
+        className="sidebar__link fw-medium p-4 dark-font"
+      >
         Settings
-      </Link>
-    </div>
```

Comment:
If you're repeating the components here like this, you could just treat it as a loop

```js
[
  ["/experiment-feature-config", "Feature Configuration Enrollment"],
  ...
].map((path, label) => (
        <Nav.Link
        as={Link}
        to={path}
        className="sidebar__link fw-medium p-4 dark-font"
      >
        {label}
      </Link>
));
```